AutomatedTesting/Levels/Multiplayer/SimpleNetworkLevelEntity/SimpleNetworkLevelEntity.prefab

```diff
@@ -485,16 +562,15 @@
                     "$type": "EditorScriptCanvasComponent",
                     "Id": 10272449525230713408,
                     "m_name": "SimpleNetworkLevelEntity.scriptcanvas",
-                    "runtimeDataIsValid": true,
                     "runtimeDataOverrides": {
                         "source": {
                             "id": "{C8F17F94-1225-5FFB-A89F-7C5546FF9DD2}",
-                            "path": "C:/prj/o3de/AutomatedTesting/Levels/Multiplayer/SimpleNetworkLevelEntity/SimpleNetworkLevelEntity.scriptcanvas"
+                            "path": "/home/gene/prj/o3de/AutomatedTesting/Levels/Multiplayer/SimpleNetworkLevelEntity/SimpleNetworkLevelEntity.scriptcanvas"
                         }
                     },
                     "sourceHandle": {
                         "id": "{C8F17F94-1225-5FFB-A89F-7C5546FF9DD2}",
-                        "path": "C:/prj/o3de/AutomatedTesting/Levels/Multiplayer/SimpleNetworkLevelEntity/SimpleNetworkLevelEntity.scriptcanvas"
+                        "path": "/home/gene/prj/o3de/AutomatedTesting/Levels/Multiplayer/SimpleNetworkLevelEntity/SimpleNetworkLevelEntity.scriptcanvas"
```

Comment:
This better still work on others system. :P

Reply:
good eye!